tests/cla_ext_test.sh

```diff
@@ -27,44 +27,51 @@ PORT_NODE4=3003
 PORT_NODE5=3004
 
 #refuse false
+#expected no packet refusal
 OUT_NODE1=$(mktemp /tmp/node1.XXXXXX)
 #DB3="-W /tmp/node3 -D sled"
 #DB3="-W /tmp/node3 -D sneakers"
-$DIR/../target/$TARGET/dtnd -d -j5s -i0 -w $PORT_NODE1 -C tcp:port=4224 -e incoming -r flooding -n node1 -s tcp://127.0.0.1:4225/node2 $STATUS_REPORTS 2>&1 &> $OUT_NODE1 &
+$DIR/../target/$TARGET/dtnd --keep-bundles -d -j5s -i0 -w $PORT_NODE1 -C tcp:port=4224 -e incoming -r flooding -n node1 -s tcp://127.0.0.1:4225/node2 $STATUS_REPORTS 2>&1 &> $OUT_NODE1 &
```

Comment:
confused because all but the second-last node should already keep the bundles. flooding always says, keep bundles unless its the final destination